src/In.ProjectEKA.HipService/DataFlow/DataEntryFactory.cs

```diff
@@ -76,29 +80,29 @@ public DataEntryFactory()
             return Option.Some(new EncryptedEntries(processedEntries.AsEnumerable(), keyMaterial));
         }
 
-        private Entry StoreComponentAndGetLink(Entry componentEntry, string transactionId)
+        private Entry StoreComponentAndGetLink(Entry componentEntry, string transactionId,string careContextReference)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unused method parameter 'transactionId'.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5414977)

Reply:
done